lib/tmuxinator/cli.rb

```diff
@@ -77,19 +77,21 @@ def create_project(name, custom_name, cli_options)
           end
         end
 
-        options[:custom_name] = custom_name
+        options[:custom_name] = options[:name]
 
-        project = Tmuxinator::Config.validate(name, options)
+        project = Tmuxinator::Config.validate(name, options, args)
         project
       end
     }
 
-    desc "start [PROJECT] [SESSION_NAME]", COMMANDS[:start]
+    desc "start [PROJECT] [ARGS]", COMMANDS[:start]
     map "s" => :start
     method_option :attach, :type => :boolean, :aliases => "-a", :desc => "Attach to tmux session after creation."
+    method_option :name, aliases: "-n",
+      :desc => "Give the session a different name"
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use the new Ruby 1.9 hash syntax.